Description: Install udev rules in the correct location
Author: Jakob Ketterl <jakob.ketterl@gmx.de>

* udev files should not go into bindir. This patch sets the correct path.
* also gets rid of hooks to create linux groups, this is done by postinst.
--- a/Makefile.am
+++ b/Makefile.am
@@ -76,14 +76,8 @@
 # sudo udevadm control --reload-rules
 #
 udevrule_DATA = $(top_srcdir)/95-perseus.rules
-udevruledir = $(bindir)
+udevruledir = $(libdir)/udev/rules.d/
 EXTRA_DIST = $(top_srcdir)/95-perseus.rules README.Windows $(top_srcdir)/build-aux/git-version-gen fpga_data.c
-
-install-data-hook:
-	-cp $(bindir)/95-perseus.rules /etc/udev/rules.d/ ; groupadd perseususb ; echo $(INST_USER) ;usermod -a -G perseususb $(INST_USER) ; udevadm control --reload-rules
-
-uninstall-hook:
-	-rm /etc/udev/rules.d/95-perseus.rules ; groupdel perseususb ; udevadm control --reload-rules
 endif
 
 #
